packages/makedumpfile/makedumpfile.spec

```
@@ -0,0 +1,50 @@
+Name: %{_cross_os}makedumpfile
+Version: 1.6.8
+Release: 1%{?dist}
+Summary: Tool to create dumps from kernel memory images
+License: GPL-2.0-or-later
```

There are some files like `arch/ppc.c` that don't have the "any later version" text.

```suggestion
License: GPL-2.0-or-later AND GPL-2.0-only
```